itests/org.openhab.core.thing.tests/src/main/java/org/openhab/core/thing/binding/BindingBaseClassesOSGiTest.java

```diff
@@ -103,11 +107,22 @@
 
     private @NonNullByDefault({}) ManagedThingProvider managedThingProvider;
     private @NonNullByDefault({}) ThingRegistry thingRegistry;
+    private @NonNullByDefault({}) ConfigurationAdmin configurationAdmin;
 
     private @Mock @NonNullByDefault({}) ComponentContext componentContextMock;
 
     @BeforeEach
-    public void beforeEach() {
+    public void beforeEach() throws IOException {
+        configurationAdmin = getService(ConfigurationAdmin.class);
+        assertNotNull(configurationAdmin);
+
+        LocaleProvider localeProvider = getService(LocaleProvider.class);
+        assertThat(localeProvider, is(notNullValue()));
+
+        new DefaultLocaleSetter(configurationAdmin).setDefaultLocale(Locale.ENGLISH);
+        waitForAssert(() -> assertThat(localeProvider.getLocale(), is(Locale.ENGLISH)));
+        waitForAssert(() -> assertThat(localeProvider.getLocale(), is(Locale.ENGLISH)));
```

Comment:
Is there a reason why this line is duplicated?

Reply:
No, copy&paste error. I‘ll remove it.